### ISSUE TITLE
Inherited Control Scenes Cannot Reset Transform Properties to Base Scene's Values

### ISSUE THREAD

**Issue by Spekel** [godotengine/godot#69226]
### Godot version

4.0 Beta 6

### System information

Windows 10

### Issue description

After changing the transform properties of an inherited control scene in the editor, the properties in the transform tab (position, size, etc...) cannot be reset to the base scene's values. Pressing the reset property button simply sets the position and size to zero.

### Steps to reproduce

1. Create a 2D scene with a control node as the root.
2. Save then create a new scene that inherits from the base scene.
3. Change the size and position of the control node in the inherited scene in the property editor. (layout -> transform)
4. Press the reset property button next to size and position.

### Minimal reproduction project

N/A

**Comment by Calinou** [comment]
cc @YuriSizov 

This may be related to Control layout changes in the inspector.

**Comment by YuriSizov** [comment]
Yes, it's kind of related, but more of a shortcoming of the current system. So size and position are never saved to disk, they never were. They are computed values for user's convenience, but the data that we save is always anchors and offsets (for anchored controls, not even that for children of containers).

When you use scene inheritance, the editor refers to the stored properties to get the fallback value. Since we don't show offsets or anchors in the most basic case anymore, it doesn't even attempt to look up those. But we do show size and position, and so we attempt to look up their fallback values. The way it works is the PackedScene resource of the base scene is inspected, and naturally, it doesn't contain any info on the size or position. So the inspector returns empty-handed.

I think this should be the case in 3.x as well, though I don't use inherited scenes to be sure. There are two ways to solve it, both don't seem right though. We either need to instance each PackedScene in the inheritance chain to check computed properties. Or we need to store them alongside real properties in scenes. Not really sure which way to go with that yet.

**Comment by pc9098** [comment]
This still happens in 4.2.2

**Comment by SerafinDelfin** [comment]
Still happens in 4.4.1
